docs/Rootless.md

```diff
@@ -33,19 +35,32 @@ endif
 ```
 
 - The iOS 14 arm64e ABI mentioned in [arm64e Deployment](arm64e-Deployment.html) is now *required* for the relevant devices
-    - Currently, it's only possible to build for the new ABI on macOS
-        - If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
+    - Currently, it's only possible to build for the new ABI on macOS (as only the Xcode tools can build for it)
+        - Unfortunately, new toolchain does not support building for the old ABI. If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
     - That being said, developers on other platforms can circumvent this *if necessary*:
-        - By adding `oldabi` as a dependency to their package (preferably only as a last resort)
+        - By [using GitHub Actions to compile your tweaks](https://github.com/p0358/SilentScreenshots/blob/master/.github/workflows/build.yml) and upload them as downloadable artifacts (free for both public and private repos)
+        - By using a macOS virtual machine
+            - On Linux you can use [OSX-KVM](https://github.com/kholia/OSX-KVM) script set
+            - Without working App Store, download full Xcode xip archive and extract it, afterwards move `Xcode.app` to `Applications` folder (you can do this graphically through Finder)
```

Comment:
Given that this is part of the OSX Theos install process, I don't think it's necessary to add here.

Reply:
The install docs for macOS just mention that you ought to have Xcode installed, they don't mention how and instead give you the link to App Store. And that isn't going to work on the VM (unless you do extra work to get Apple ID set up, which is risky and not really worth it here when there's alternative).

So I thought mentioning this, as a difference, is worth it. The alternative would be to update the macOS install docs to mention you can install Xcode from either app store or with xip archive.

Reply:
I think the ways one can install Xcode is out of scope for the Theos docs...all we care about is that it's required. It may be worth swapping the Xcode hyperlink in the macOS doc to the Apple developer site link, however, as 1) it's faster than the App Store and 2) follows with what you mention about decreased risk.

Reply:
I swapped the link to https://developer.apple.com/download/applications/ as it has links to both then:
![image](https://github.com/theos/theos.dev/assets/5182588/9456c1ce-81ec-44ef-a8d7-f58654ff9a4f)